styles/main.css

```diff
@@ -5,13 +5,13 @@ body {
 
 h1 {
     color: #080C3B;
-    background-image:url(/CSD120/images/BG.png);
+    background-image:url(../images/BG.png);

```

Comment:
This works, but it's recommended to use relative urls when possible as it's less likely to break if your project ever moves servers